docs/source/conf.py

```diff
@@ -385,6 +385,12 @@ def generate_weights_table(module, table_name, metrics, include_pattern=None, ex
 generate_weights_table(
     module=M.detection, table_name="detection", metrics=[("box_map", "Box MAP")], exclude_pattern="Keypoint"
 )
+generate_weights_table(
+    module=M.detection,
+    table_name="instance_segmentation",
+    metrics=[("box_map", "Box MAP"), ("mask_map", "Mask MAP")],
+    include_pattern="Mask",
```

Comment:
This seems hack way to add docs by `include_patterns` and `exclude_pattern`. Not sure if we could come up with better way?

Reply:
TBF, this entire doc generation strategy is a big hack.

We discussed alternatives in https://github.com/pytorch/vision/pull/5933#discussion_r863857591. Regexs are an alternative but they're potentially more complex, especially now that we need to ignore more than one pattern.

Ultimately if we have a registration mechanism for all these models and weights, we might be able to remove this.

Reply:
Yeah I'm too not in support of regex. Its like virus.